tests/auth.setup.ts

```
@@ -36,5 +40,21 @@ setup("Authentication as a PRO user", async ({ request }) => {
+
+setup("Authentication as a PRO user", async ({ request }) => {
+  process.env["ACCESS_TOKEN"] = await getAccessToken(
+    process.env.EMAIL_PRO!,
+    process.env.PASSWORD_PRO!,
+    request
+  );
+});
+
+setup("Authentication as an user with no rights", async ({ request }) => {
+  process.env["ACCESS_TOKEN_USER_NO_RIGHTS"] = await getAccessToken(
+    process.env.EMAIL!,
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)